pocs/tests/test_dispatch_scheduler.py

```diff
@@ -82,6 +98,45 @@ def test_get_observation(scheduler):
     assert isinstance(best[1], float)
 
 
+@pytest.fixture
+def temp_file():
+    temp_file = 'temp'
+    with open(temp_file, 'w') as f:
+        f.write('')
+
+    yield temp_file
+    os.unlink(temp_file)
+
+
+def test_get_observation_reread(field_list, observer, temp_file):
+    time = Time('2016-08-13 10:00:00')
+
+    # Write out the field list
+    temp_field_file = temp_file
```

Comment:
Why bother with _renaming_ the variable?

Reply:
Remnant of when I was doing it a different way. Fixed. Thanks.